tutorials/AgileRL/agilerl_maddpg.py

```diff
@@ -154,16 +154,31 @@
     # Training loop
     for idx_epi in trange(max_episodes):
         for agent in pop:  # Loop through population
-            state = env.reset()[0]  # Reset environment at start of episode
+            state, info = env.reset()  # Reset environment at start of episode
             agent_reward = {agent_id: 0 for agent_id in env.agents}
             if INIT_HP["CHANNELS_LAST"]:
                 state = {
                     agent_id: np.moveaxis(np.expand_dims(s, 0), [3], [1])
                     for agent_id, s in state.items()
                 }
             for _ in range(max_steps):
-                action = agent.getAction(state, epsilon)  # Get next action from agent
-                next_state, reward, termination, truncation, _ = env.step(
+                agent_mask = info["agent_mask"] if "agent_mask" in info.keys() else None
```

Comment:
What is an agent mask? Never heard of it. Some envs have action masks in info so maybe it’s a typo?

Reply:
Agent masking is used when you don't want to take a new action for a certain agent. See our docs here: https://docs.agilerl.com/en/latest/api/algorithms/matd3.html#agent-masking
It won't interfere with PettingZoo, but is important for environments when agents don't always take concurrent actions.

Reply:
Good to know, sounds cool